Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -339,11 +339,11 @@ macro usePreviousFrame()
     end
 end
 
-macro reloadMemoryRegistersFromInstance(instance, scratch1, scratch2)
+macro reloadMemoryRegistersFromInstance(instance, scratch1)
 if not ARMv7
     loadp Wasm::Instance::m_cachedMemory[instance], memoryBase
     loadp Wasm::Instance::m_cachedBoundsCheckingSize[instance], boundsCheckingSize
```

Comment:
`boundsCheckingSize` looks unused.

Reply:
It is callee-save register used in wasm code, so loading this is necessary.